website/common/locales/en/groups.json

```diff
@@ -146,6 +146,7 @@
   "cannotSendGemsToYourself": "Cannot send gems to yourself. Try a subscription instead.",
   "badAmountOfGemsToSend": "Amount must be within 1 and your current number of gems.",
   "report": "Report",
+  "flagged": "Flagged",
```

Comment:
Could you change the key to `hasBeenFlagged` just to make it easier for us to grep for it? Grepping for single-word keys results in a lot of false positives. :)  I think the only place it's used is line 4 of website/client/components/challenges/challengeItem.vue